### DISCUSSION TITLE
Mensagem de erro

### DISCUSSION THREAD

**Discussion by diogo-alves13** [arkypita/LaserGRBL#1695] [Q&A]
Eu uso a placa MakerBase DLC32 e as vezes está aparecendo mensagem de erro dizendo que "LaserGrbl encontrou um problema com a placa" e a causa é " parou de responder" e ai a máquina para de se movimentar e o Laser fica queimando parado no mesmo lugar.

![IMG-20220101-WA0007~3](https://user-images.githubusercontent.com/96754838/147863240-44b11b11-afa5-490e-ae9c-486bfd35d85c.jpg)

 Alguém sabe me dizer qual pode ser a causa

**Comment by edsonee** [comment]
mira el consejo 1
https://youtu.be/XuFWCh3G-Mg

**Comment by edsonee** [comment]
usa LASERGRBL 4.63 or 4.7
